source/end-user-guide/access/acceptable-use-policy-and-privacy-policy.rst

```diff
@@ -0,0 +1,53 @@
+Acceptable Use Policy and Privacy Policy
+=========================================
+
+To use Mattermost, all users **must agree** to the Mattermost **Acceptable Use Policy** and **Privacy Policy**. You cannot create or use a Mattermost account unless you agree to both policies. The Acceptable Use Policy and Privacy Policy may change over time, and all users are required to accept and comply with the most current versions.
+
+Acceptable Use Policy
+---------------------
+
+The **Acceptable Use Policy (AUP)** explains how Mattermost may and may not be used.
```

Comment:
Are we using this AUP acronym anywhere? If not, probably don't need to show it here